llvm/utils/TableGen/SearchableTableEmitter.cpp

```diff
@@ -638,6 +662,7 @@ void SearchableTableEmitter::collectTableEntries(
 
   Record *IntrinsicClass = Records.getClass("Intrinsic");
   Record *InstructionClass = Records.getClass("Instruction");
+  bool SawInstructionField = false;
```

Comment:
SawInstructionField->HasInstructionField?

Reply:
I have removed this variable since it is no longer necessary due to having moved the map population logic.